internal/otelcollector/config/metric/agent/receivers.go

```diff
@@ -47,25 +39,16 @@ func makeKubeletStatsConfig() *KubeletStatsReceiver {
 }
 
 func makePrometheusSelfConfig() *PrometheusReceiver {
-	targets := []*promtargetgroup.Group{
-		{
-			Targets: []prommodel.LabelSet{
-				{
-					prommodel.AddressLabel: prommodel.LabelValue(fmt.Sprintf("${%s}:%d", config.EnvVarCurrentPodIP, ports.Metrics)),
-				},
-			},
-		},
-	}
-
 	return &PrometheusReceiver{
-		Config: promconfig.Config{
-			ScrapeConfigs: []*promconfig.ScrapeConfig{
+		Config: PrometheusConfig{
+			ScrapeConfigs: []ScrapeConfig{
 				{
-					JobName:          "opentelemetry-collector",
-					ScrapeInterval:   prommodel.Duration(10 * time.Second),
-					HTTPClientConfig: promcommonconfig.DefaultHTTPClientConfig,
```

Comment:
is this enabled by defualt so this is removed ?

Reply:
Yes, those are default values